setup.cfg

```diff
@@ -44,6 +44,7 @@ dev =
     # https://github.com/docker/docker-py/issues/2902
     pytest-docker==0.10.3; python_version < '3.10' or sys_platform != 'win32'
     pylint==2.11.1
+    mock>=4.0.0
```

Comment:
`mock` is needed for backport of `AsyncMock` in py3.7 (only available in the stdlib `unittest.mock` in 3.8+)